### DISCUSSION TITLE
elsa 3 - where i can find repo that same as in docker hub ?

### DISCUSSION THREAD

**Discussion by nurhidayat-agung** [elsa-workflows/elsa-core#5562] [Q&A]
hello elsa, 
ehere i can find and clone repositories that same as in docker hub ?
because i find different version both elsa server and studio betwen nuget or documentation and in docker hub
thx

**Comment by glime-ai** [comment]
You can find and clone the repositories for Elsa Server and Studio on GitHub. The official Elsa GitHub repository is where the source code is maintained, and it typically aligns with the Docker images available on Docker Hub. However, it's important to note that the versions on NuGet or in the documentation may differ from those on Docker Hub because Docker images are built and pushed at specific points in time, and the code in the GitHub repository continues to evolve.

To clone the repositories, you can follow these steps:

1. Visit the Elsa Workflow GitHub repository: [https://github.com/elsa-workflows/elsa-core](https://github.com/elsa-workflows/elsa-core)
2. Click on the "Code" button and then choose "Clone" to clone the repository using your preferred method (HTTPS, SSH, or GitHub CLI).
3. If you want to clone using HTTPS, you can use the following command in your terminal:
   ```
   git clone https://github.com/elsa-workflows/elsa-core.git
   ```
4. Once cloned, you can check out the specific branch or tag that corresponds to the version you are interested in.

Keep in mind that the Docker images on Docker Hub may be built from specific branches or tags, so if you want to match the Docker image version, you should check out the corresponding branch or tag in the GitHub repository.

For the latest Docker images, you can always pull them from Docker Hub using the following commands:
- For Elsa Server + Studio:
  ```
  docker pull elsaworkflows/elsa-server-and-studio-v3:latest
  ```
- For Elsa Server:
  ```
  docker pull elsaworkflows/elsa-server-v3:latest
  ```
- For Elsa Studio:
  ```
  docker pull elsaworkflows/elsa-studio-v3:latest
  ```

If you need further assistance with a specific version or have more questions about the repositories, please let me know!

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
